en/identity-server/next/docs/deploy/security/encrypt-passwords-with-cipher-tool.md

```diff
@@ -8,9 +8,14 @@ The instructions on this page explain how plain text passwords in configuration
 
 ## Encrypt passwords
 
-To encrypt passwords on the WSO2 Identity Server:
+To encrypt passwords on the WSO2 Identity Server, you can use either asymmetric or symmetric encryption.
```

Comment:
Add an important note mentioning that symmetric is recommended due to its quantum security.
And have the symmetric encryption section first.